.github/workflows/build_and_test_check.yml

```diff
@@ -16,6 +16,8 @@ jobs:
           distribution: 'adopt'
 
       - name: Build with Gradle
+        env:
+          ONESIGNAL_API_KEY: ${{ secrets.ONESIGNAL_API_KEY }}
```

Comment:
Už by sme to mali mať v systémovej premennej z terraformu(infrastructure/aws/ecs/backend.tmpl:42) . Ale ak to ide dať takto jednoducho, určite preferujem tento spôsob.

Reply:
Ano, funguje. ;-)

Reply:
V tom prípade Česko digital fail #88 🤦 .

Reply:
V tom prípade hurá 👍 .